chart/identity-manager-sts/templates/ingress.yaml

```diff
@@ -0,0 +1,36 @@
+---
+{{- if .Values.ingress.enabled -}}
+{{- $paths := .Values.ingress.paths -}}
+{{- $fullName := include "common.names.fullname" . -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
```

Comment:
This should be a variable. I would wrap this block in a `with` clause. Letsendrypt-prod is specific to our clusters.